django/conf/global_settings.py

```diff
@@ -502,6 +502,17 @@
 CACHE_MIDDLEWARE_SECONDS = 600
 CACHE_MIDDLEWARE_ALIAS = 'default'
 
+#########
+# TASKS #
+#########
+
+# The task backends to use.
+QUEUES = {
```

Comment:
What about `TASKS`? We have `django.db > DATABASES` and  `django.core.cache > CACHES`, so `django.tasks > TASKS` works well.